### ISSUE TITLE
Automated code correction

### ISSUE THREAD

**Issue by cEhrnsberger** [madskristensen/WebAnalyzer#6]
Is there any plan to add automated code correction (on save for example) as an optional feature?
I mean red squiggly would help a lot for the moment but automated correction would be the killer feature!

**Comment by madskristensen** [comment]
Yeah, I was considering it. ESLint does provide fixes that Web Analyzer can consume and apply. That would be cool.

Red squigglies are easy to add but hard to keep updated correctly so they don't jump around as changes are made to the document.

**Comment by cEhrnsberger** [comment]
I also liked the way the old WebEssentials (until 2013) displayed the JSHint Warnings when hovering above the files in solution explorer. I think this is more convenient to use and less obstrusive compared to the error list. I think this could be an alternative/first step to red squiggly in the code. 
We are currently using another tool for code inspection but its options are limited compared to ESLint, so i would love to switch to Web Analyzer for this purpose completely.
Please keep up the excellent work on this extension! :+1:

**Comment by madskristensen** [comment]
Web Essentials didn't show JSHint warnings when hovering above the file in Solution Explorer. That must have been Productivity Power Tools that did that when the file contained errors in the Error List

**Comment by cEhrnsberger** [comment]
Oh you are right, but it doesn't seem to work anymore. At least for me.

**Comment by madskristensen** [comment]
Don't know why that is. Must be a change to the Productivity Power Tools
